sympy/printing/latex.py

```diff
@@ -740,13 +740,14 @@ def _hprint_Function(self, func):
         Logic to decide how to render a function to latex
           - if it is a recognized latex name, use the appropriate latex command
           - if it is a single letter, just use that letter
+          - if it is a single letter before ^ or _ inserted by _deal_with_super_sub
```

Comment:
Either add ", just use that letter" or write this and the bullet above as a single bullet.

Reply:
Probably, "if it is a single letter, excluding sub- and superscripts, just use that letter" could do.